### PR TITLE
fix: social image and add keywords

### PR DESCRIPTION
**Pull request by jainpawan21**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
    fix: social image and add meta keywords for seo
- **What is the current behavior?** (You can also link to an open issue here)
   social preview image is broken
- **What is the new behavior (if this is a feature change)?**
  fixed
- **Other information**:
  for robots metadata see [this](https://blog.johnnyreilly.com/2021/10/18/docusaurus-meta-tags-and-google-discover)

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -39,7 +39,15 @@ const codeTheme = require('./src/utils/prism');
...
-        image: '/img/social-preview.jpg',
+        metadata: [
+          { name: 'robots', content: 'max-image-preview:large' },
+          {
+            name: 'keywords',
+            content:
+              'novu,novu documentation,novu docs,notification,notification insfrastructure,open source,oss',
+          },
+        ],
+        image: 'https://novu.co/images/social-preview.jpg',
```
Comment (scopsy) [inline docs/docusaurus.config.js:50]:
```suggestion
        image: '/img/social-preview.jpg',
```

this asset is also available on the docs project, any specific reason you wanted. to fetch it form the landing page?

Comment (jainpawan21) [reply docs/docusaurus.config.js:50]: right now it is not working, showing 404 😅

Comment (jainpawan21) [reply docs/docusaurus.config.js:50]: If we share doc link, image is not showing because this asset is missing

Comment (jainpawan21) [reply docs/docusaurus.config.js:50]: Thanks Andrew, I missed this

**Review by scopsy — APPROVED** [review]
Thank you @jainpawan21 ❤️

**Pull request merged**